src/index.ts

```diff
@@ -9,3 +8,4 @@ export {
 } from "./schema"
 export { basicSchemaAdapter } from "./basicSchema"
 export { docFromSpans, blocksFromNode } from "./traversal"
```

Comment:
I think the only thing remaining to do is rename the `docFromSpans` export to `initializeProsemirrorDoc`.

Reply:
What about `pmDocFromSpans`? Subsequently, also rename `blocksFromNode` to `pmNodeToSpans`?